application/forms/TimeRangePicker/CommonForm.php

```diff
@@ -165,7 +165,7 @@ protected function urlToForm()
                     unset($options['']);
 
                     foreach ($options as $option => $_) {
-                        if ($seconds === $option * $factor) {
+                        if ($seconds === (int) $option * $factor) {
```

Comment:
This casts only `$option`. Is this intended?

Reply:
Yes! Because `$option` is an array key, that's why it must be explicitly converted to int.